site/docs/plugins/session.md

```diff
@@ -326,6 +326,46 @@ If you must use the option (which is of course still possible), you should know
 Make sure you understand the consequences of this configuration by reading [this](../guide/deployment-types.md) article and especially [this](./runner.md#sequential-processing-where-necessary) one.
 :::
 
+### Chat Migrations
+
+If you are using sessions for groups, you should be aware that Telegram migrates regular groups to supergroups under certain circumstances (e.g. [here](https://github.com/telegramdesktop/tdesktop/issues/5593)).
+
+This migration can only occur once per group chat but it can cause inconsistencies.
+This is because the migrated chat is technically a completely different chat with a different chat identifier and hence a different session identifier.
```

Comment:
```suggestion
This is because the migrated chat is technically a completely different chat that has a different chat identifier, and hence a different session identifier.
```